### ISSUE TITLE
extreme hills biome doesn't exist

### ISSUE THREAD

**Issue by alexHunter854** [DarkPacks/SevTech-Ages#4353]
## Bug Description

at age five you need to find emeralds in the extreme hills biome, for me atleast, the nature's compass didn't find it.
if the emerald ore spawining biome the advancment doesn't reflect that

## Steps to Reproduce (for bugs)

1. create new world
2. cheat in or make a nature's compass
3. search for extreme hills
4. "status: not found"

## Logs

* Client/Server Log:there are no logs
* Crash Log:

## Client Information
* Modpack Version: SevTech: Ages of the Sky: Version: 3.2.0-alpha.4-skyblock
* Java Version:1.8.0_51
* Launcher Used: twitch launcher
* Memory Allocated: 6350 MB
* Server/LAN/Single Player: Single Player
* Optifine Installed: no
* Shaders Enabled: no

## World Information
* Modpack Version world created in: SevTech: Ages of the Sky: Version: 3.2.0-alpha.2-skyblock
* Additional Content Installed: Integrated Tunnels, a Cyclopse Core update and a Common Capabilities update

**Comment by Heylan123** [comment]
Emerald Ore can be found in the extreme hills + which can be found on the nature compass

**Comment by sam-kirby** [comment]
> Emerald Ore can be found in the extreme hills + which can be found on the nature compass

This is correct - the only extreme hills variant that is set to spawn is `minecraft:extreme_hills_with_trees` biome. This is also known as Extreme Hills +. We should remove more of the unused biomes from the Nature's Compass.
